source/delegate-factory/deploys.json

```diff
@@ -1,4 +0,0 @@
-{
-  "1": "0xA55523eeD98Ba64fF7673b22963DF34E3A0305FE",
```

Comment:
do we want to keep these in the delegate? Seems like something we shouldnt lose

Reply:
That's right, good call. Though this file should indicate what is actually deployed. To discuss

Reply:
Added some README content to address.